documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,45 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.
+
+To consume the ApiKeyValueMapUpload step, proceed as follows:
+
+* Get the SAP API management service key from the SAP BTP sub account cockpit, under instance and subscriptions for service API Management, API portal under apiportal-apiaccess plan.
+* Store your service key created for SAP API Management in the the Jenkins server as a secret text.
+* Create a new Jenkins pipeline designated for the ApiKeyValueMapUpload step.
+* Execute the pipeline and validate step exection results as explained in the blog [Integration Suite Piper commands](https://blogs.sap.com/2022/01/05/orking-with-integration-suite-piper-commands/)
+
```

Comment:
Why the new line here, does that do anything to the resulting documentation, and if so: is that intended?

Reply:
removed